### PR TITLE
Getting started

### PR DESCRIPTION
**Pull request by julien51**

### CODE REVIEW

**Comment by julien51** [comment]
I think we need to remove the "Using etherscan" article and move it to the guides.

File: _redirects [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Redirects from what the browser requests to what we serve
```
Comment (julien51) [inline _redirects:1]: Let's put this file in another PR so we can merge this one and that the redirects ready for when we can ship!

**Review by julien51 — COMMENTED** [review]
This is looking 🔥 !

A few more things before we can ship it!

* Let's make sure we add a redirect for any file we delete like `/basics/new-to-unlock/deploying-a-lock.md` and any file we moved like `/new-to-web3/ethereum-architecture.md` inside the docusaurus config file
* Let's keep the Guides changes in another PR so we can merge this "getting started" without impacting the rest.
* When we remove stuff from docusaurus.config.js let's make sure we still have a redirect in the `_redirects` file otherwise we will have 404

File: _redirects (hunk 1 above)
Comment (wonderwomancode) [reply _redirects:1]: Left the redirects that pointed to existing articles. Moved the rest over to move-to-guides branch and opened a pull request there with a note to wait until after guides launch.

**Comment by wonderwomancode** [comment]
> 

Added the client-side redirects. Put the guides back in and moved the server-side redirects to the new pull requests in move-to-guides branch, added stuff back into docusaurus.config.js since I added the guides stuff back in.

**Review by julien51 — COMMENTED** [review]
Looking good!

**Review by julien51 — COMMENTED** [review]
LGTM

**Review by wonderwomancode — APPROVED** [review]
